### PR TITLE
pgcli: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by iMichka** [comment]
```
==> python -c import setuptools... --no-user-cfg install --prefix=/home/linuxbrew/.linuxbrew/Cellar/pgcli/1.9.0/libexec/vendor --single-version-externally-managed --record=installed.txt
.
Last 150 lines from /home/linuxbrew/.cache/Homebrew/Logs/pgcli/01.python:
2018-03-08 11:15:47 +0000

python
-c
import setuptools, tokenize
__file__ = 'setup.py'
exec(compile(getattr(tokenize, 'open', open)(__file__).read()
  .replace('\r\n', '\n'), __file__, 'exec'))
--no-user-cfg
install
--prefix=/home/linuxbrew/.linuxbrew/Cellar/pgcli/1.9.0/libexec/vendor
--single-version-externally-managed
--record=installed.txt

Failed to execute: python

==> Formula
```

File: Formula/pgcli.rb [hunk 1]
```diff
@@ -94,15 +94,15 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
     resources.each do |r|
       r.stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+        system *(OS.mac? ? "python" : "python2"), *Language::Python.setup_install_args(libexec/"vendor")
```
Comment (sjackman) [inline Formula/pgcli.rb]: I'd recommend instead using
```ruby
system "python2.7", *Language::Python.setup_install_args(libexec/"vendor")
```
Joe has said that we would accept these changes upstream.

File: Formula/pgcli.rb [hunk 2]
```diff
@@ -94,15 +94,15 @@ def install
...
       end
     end
 
     # backports is a namespace package
     touch libexec/"vendor/lib/python2.7/site-packages/backports/__init__.py"
 
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-    system "python", *Language::Python.setup_install_args(libexec)
+    system *(OS.mac? ? "python" : "python2"), *Language::Python.setup_install_args(libexec)
```
Comment (sjackman) [inline Formula/pgcli.rb]: Ditto.

File: Formula/pgcli.rb (hunk 1 above)
Comment (iMichka) [reply Formula/pgcli.rb]: Yes, he told me that. Let's go for that solution. It fixes their build for snow leopard I think.

**Comment by iMichka** [comment]
Abandoning in favour of https://github.com/Homebrew/homebrew-core/pull/25027

**Pull request closed without merging**